packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1316,9 +1325,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                         formExponentialFn({ coefficient: 20, initialDelay: 0 }),
                     ),
                     {
-                        initialDelayMs: this.runtimeOptions.summaryOptions.initialSummarizerDelayMs,
+                        initialDelayMs: currentInitialSummarizerDelayMs,
                     },
-                    this.runtimeOptions.summaryOptions.summarizerOptions,
+                    this.runtimeOptions.summaryOptions.summaryConfigOverrides?.state === "disableHeuristics" ?? false,
```

Comment:
Does this work like `this.runtimeOptions.summaryOptions.summaryConfigOverrides ? this.runtimeOptions.summaryOptions.summaryConfigOverrides?.state === "disabledHeuritsitcs" : false`? I've never seen this nullish coalescing pattern before.

Reply:
It was actually wrong, I think we need to use: this.summaryConfiguration.state === "disableHeuristics"  
I created a property similar to what we did for the initialSummarizerDelayMs  

It will allow us to add some unit tests.